### DISCUSSION TITLE
hello: how to remove public key from my set or computer completely

### DISCUSSION THREAD

**Discussion by ghost** [community/community#36738] [Apps, API and Webhooks]
how to remove public key from my set or computer compeletly

**Comment by ProDCG** [comment]
If you're talking about SSH keys, if you go to 

`Settings (top right) -> SSH and GPG keys` you can then go ahead and delete and keys you don't want to use anymore.

**Comment by ossanna16** [comment]
Hey there @ghost and thank you for asking a good question! I'm glad you found our community 🙂
